doc/manual/parallel-computing.rst

```diff
@@ -799,18 +801,18 @@ starting workers on the same host, is implemented::
 
 The :func:`launch` method takes the following arguments:
 
-- ``manager::ClusterManager`` - the cluster manager :func:`addprocs` is called with
-- ``params::Dict`` - all the keyword arguments passed to :func:`addprocs`
-- ``launched::Array`` - the array to append one or more ``WorkerConfig`` objects to
-- ``c::Condition`` - the condition variable to be notified as and when workers are launched
+- ``manager::ClusterManager``: the cluster manager that :func:`addprocs` is called with
+- ``params::Dict``: all the keyword arguments passed to :func:`addprocs`
+- ``launched::Array``: the array to append one or more ``WorkerConfig`` objects to
+- ``c::Condition``: the condition variable to be notified as and when workers are launched
 
 The :func:`launch` method is called asynchronously in a separate task. The termination of this task
 signals that all requested workers have been launched. Hence the :func:`launch` function MUST exit as soon
 as all the requested workers have been launched.
 
-Newly launched workers are connected to each other, and the master process, in a all-to-all manner.
+Newly launched workers are connected to each other, and the master process, in an all-to-all manner.
 Specifying the command argument ``--worker <cookie>`` results in the launched processes initializing themselves
-as workers and connections being setup via TCP/IP sockets. Optionally, ``--bind-to bind_addr[:port]``
+as workers and connections being set up via TCP/IP sockets. Optionally, ``--bind-to bind_addr[:port]``
```

Comment:
Depends on usage, I think this referring to the state reads better as "being set up" than "being setup."